programs/bubblegum/program/src/lib.rs

```diff
@@ -888,6 +949,194 @@ fn process_collection_verification<'info>(
     )
 }
 
+fn assert_signed_by_tree_delegate(
+    tree_config: &TreeConfig,
+    incoming_signer: &Signer,
+) -> Result<()> {
+    require!(
+        incoming_signer.key() == tree_config.tree_creator
+            || incoming_signer.key() == tree_config.tree_delegate,
+        BubblegumError::TreeAuthorityIncorrect,
+    );
+    Ok(())
+}
+
+fn fetch_old_metadata_args(
```

Comment:
Perhaps rename to `fetch_current_metadata_arg`.

Reply:
done